generators/app/index.js

```diff
@@ -49,6 +49,7 @@ module.exports = Generator.extend({
 
   default: function () {
     var cmd = [
+      'rm -f master.tar.gz pattern-lab-starter-master patternlab',
```

Comment:
See line 77 as well for a place where some cleanup may need to happen on a replay. If themePath isn't set to the default `./` an error is hit trying to move the pattern lab directory into the right spot.:

```
  try {
         exec('mv patternlab "' + dest + '"', {
           encoding: 'utf8'
         });
       } catch (error) {
         console.error('Could not move theme into themePath.');
       }
```